src/components/KInput/KInput.cy.ts

```diff
@@ -188,11 +188,11 @@ describe('KInput', () => {
 
     cy.get('.k-input .mask-value-toggle-button').should('be.visible')
     cy.get('.k-input input').should('have.attr', 'type', 'password')
+    cy.get('.k-input input').focus()
     cy.get('.k-input .mask-value-toggle-button').click()
     cy.get('.k-input input').should('have.attr', 'type', 'text')
     cy.get('.k-input .mask-value-toggle-button').click()
-    cy.get('.k-input input').should('have.attr', 'type', 'password')
-
+    cy.get('.k-input input').should('have.attr', 'type', 'password').should('be.focused')
```

Comment:
Note: in real user interactions, `mouseup` moves the input cursor to the start position, but I couldn't reproduce this behavior in test cases, so this part of the test is not included.